flask_securest/userstores/file_userstore.py

```diff
@@ -0,0 +1,51 @@
+#########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+import os
+import yaml
+
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+from flask_securest.userstores import simple
```

Comment:
I think it's more elegant to do:
`from flask_securest.userstores.simple import SimpleUserstore`
and then in your class definition inherit from SimpleUserstore and not simple.SimpleUserstore